src/modules/commander/Commander.cpp

```diff
@@ -1741,7 +1741,7 @@ Commander::run()
 
 		// abort auto mode or geofence reaction if sticks are moved significantly
 		// but only if not in a low battery handling action
-		const bool low_battery_reaction = _battery_warning < battery_status_s::BATTERY_WARNING_CRITICAL;
```

Comment:
Fixes regression from #13695. Otherwise, RC override would only work during a low battery reaction, which is the opposite of the intended effect.